src/OSPSuite.Infrastructure.Import/Core/DataFormat/AbstractColumnsDataFormat.cs

```diff
@@ -94,7 +93,8 @@ private void setSecondaryColumnUnit(ColumnInfoCache columnInfos)
             foreach (var relatedColumn in columnInfos.RelatedColumnsFrom(column.Name))
             {
                var relatedParameter = mappings.FirstOrDefault(p => p.ColumnName == relatedColumn.Name);
-               if (relatedParameter != null && (relatedParameter.MappedColumn.Unit == null || relatedParameter.MappedColumn.Unit.SelectedUnit == UnitDescription.InvalidUnit))
+               if (relatedParameter != null && (relatedParameter.MappedColumn.Unit == null ||
+                                                relatedParameter.MappedColumn.Unit.SelectedUnit == UnitDescription.InvalidUnit))
```

Comment:
`relatedParameter.MappedColumn` cannot be null?

Reply:
Judging from the MappingDataFormatParameter constructor I would say no

In DataFormatParameter.cs:
```
      public MappingDataFormatParameter() 
      {
         MappedColumn = new Column();
      }
```

Reply:
This line of code is very very hard to read. If we are aleady in there, can we simplify it. Create a small method that expresses what it is. Better, this is a a method of the object itself (replated parameter)